packages/common/src/params.ts

```diff
@@ -1,15 +1,14 @@
 export const BURN_ADDRESS = "0x0000000000000000000000000000000000000000";
 
-// Rocket Pool Fee Recipient is the same for both networks (mainnet and prater)
-export const rocketPoolFeeRecipient =
-  "0xd4E96eF8eee8678dBFf4d535E033Ed1a4F7605b7";
-
 export const ROCKET_POOL_FEE_RECIPIENT =
   "0xd4E96eF8eee8678dBFf4d535E033Ed1a4F7605b7" as const;
 
 export const MEV_SP_ADDRESS_PRATER =
   "0xF21fbbA423f3a893A2402d68240B219308AbCA46" as const;
 
-// TODO: Put the mainnet address here
 export const MEV_SP_ADDRESS_MAINNET =
   "0xAdFb8D27671F14f297eE94135e266aAFf8752e35" as const;
+
+export const MAINNET_ORACLE_URL = "https://sp-api.dappnode.io" as const;
+
+export const TESTNET_ORACLE_URL = "http://65.109.102.216:7300" as const;
```

Comment:
I'd like to have a domain for the testnet oracle, it is a bit weird to have the IP and port hardcoded here, but since we don't have it it's ok...

Reply:
i can ask for a goerli domain, but for now we should either keep it like this or remove prater oracle integration